recipes/pytabix/build.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
-export LD_LIBRARY_PATH="-L$PREFIX/lib"
-export CPATH=${PREFIX}/include
```

Comment:
The compiler packages should set this up for us now.

Reply:
Thanks @mbargull that helped clean things up :)